docs/releasing.md

```diff
@@ -57,68 +55,53 @@ and that no new PRs should be merged until you are done.
 ```sh
 # Run the tests.
 bash ./lighthouse-core/scripts/release/test.sh
-# Change into the newly-created pristine folder.
-cd ../lighthouse-pristine
 ```
 
 Confirm DevTools integration will work:
 ```sh
-# Checkout the DevTools Frontend code.
-# See: https://chromium.googlesource.com/devtools/devtools-frontend/+/HEAD/docs/workflows.md
-
-# Roll to DevTools Frontend folder.
-# If at ~/src/devtools/devtools-frontend, the command to run is:
-yarn devtools ~/src/devtools/devtools-frontend
-
-# Build DevTools Frontend.
-cd ~/src/devtools/devtools-frontend
-gclient sync && autoninja -C out/Default
-
-# Verify things work in DevTools.
-# Mac:
-"/Applications/Google Chrome Canary.app/Contents/MacOS/Google Chrome Canary" --custom-devtools-frontend=file://$HOME/src/devtools/devtools-frontend/out/Default/resources/inspector
+# Change into the newly-created pristine folder.
+cd ../lighthouse-pristine
 
-# Verify that the Lighthouse panel still works. Consider the new features that have been added.
-# If anything is wrong, stop releasing, investigate, land a fix and start over.
+yarn test-devtools
 
-# (TODO: when this is a required check in CI, can remove from release steps)
-# Run webtests.
-DEVTOOLS_PATH=~/src/devtools/devtools-frontend yarn test-devtools
+# Do some manual testing on a number of sites.
+yarn open-devtools
 
 # Done with DevTools for now, will open a CL later.
-```
 
-Confirm Lightrider integration will work:
-```sh
-# See the internal README for updating Lighthouse.
+# Leave pristine folder.
+cd ../lighthouse
+```
 
-# Test things out locally, if happy, deploy to canary and see how the graphs react. 20 minutes should be enough time.
+### Lightrider
 
-# Do the stuff in "Test LR changes in Canary".
-# go/lightrider-doc#test-lr-changes-in-canary
+Confirm Lightrider integration will work.
 
-# Verify that Lightrider works properly, and is generating reports fully. Consider the new features that have been added.
-# Note: if the changes include proto changes make sure that the API has those new fields.
-# If anything is wrong, stop releasing, investigate, land a fix and start over.
+1. See the internal README for updating Lighthouse: go/lightrider-doc
+1. Roll to the canary feed in a workspace
+1. Run the tests
+1. Update/fix any failing tests
+1. All good: Hold on submitting a CL until after cutting a release
```

Comment:
```suggestion
1. All good: Hold on rolling Lighthouse until after cutting a release
```
Might be confusing if RM needs to merge CLs to update dailing tests.

Reply:
You can't update any tests without also rolling Lighthouse, so there's only gonna be one CL.